Definitely not master

kanyetehgiant opened the pull request:
hopefully this is right

kanyetehgiant committed:
component concepts

practice building to do list structures. only adds to list. no styles

kanyetehgiant committed:
Concepts-adds and removes

updated to remove tasks from queue on click and set the the item state array to the filtered item list.

kanyetehgiant committed:
Update App.js

poutler commented:
Got sidetracked tonight, will take a look tomorrow @kanyetehgiant 

You can click `files changed` underneath the PR title and make comments on specific lines. Just from a first glance, I saw a few places we can add some arrow functions to get rid of `bind()` calls, and we can probably get rid of some constructor functions (declaring state in a constructor is somewhat of an anti pattern, because you're mutating the component at run time; it breaks immutability)

kanyetehgiant committed:
ES6 Changes + Edit Comments

Got rid of bind statements by switching to arrow functions. Wrote a comment about structuring for edit component to replace the delete onClick method.
